src/modules/visitor-space/components/AutocompleteFieldFilterForm/AutocompleteFieldFilterForm.tsx

```diff
@@ -0,0 +1,105 @@
+import { yupResolver } from '@hookform/resolvers/yup';
+import { FormControl } from '@meemoo/react-components';
+import clsx from 'clsx';
+import { isNil } from 'lodash-es';
+import { type FC, type ReactNode, useEffect, useState } from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import { StringParam, useQueryParam } from 'use-query-params';
+import { object as yupObject, string as yupString } from 'yup';
+
+import AutocompleteFieldInput from '@visitor-space/components/AutocompleteFieldInput/AutocompleteFieldInput';
+import { AutocompleteField } from '@visitor-space/components/FilterMenu/FilterMenu.types';
```

Comment:
I would shorten the import path a bit, so these two components can come from the same import path. 
`import { AutocompleteFieldInput, AutocompleteField } from '@visitor-space/components'`

Reply:
that requires a barrel file
and those often have high performance impact on tests
https://marvinh.dev/blog/speeding-up-javascript-ecosystem-part-7/
https://dev.to/tassiofront/barrel-files-and-why-you-should-stop-using-them-now-bc4